Auth0.Windows/Auth0Client.cs

```diff
@@ -199,10 +205,20 @@ public Task<JObject> GetDelegationToken(string targetClientId, IDictionary<strin
         /// <summary>
         /// Log a user out of a Auth0 application.
         /// </summary>
-        public void Logout()
+        /// <param name="forceClearCache">Force clearing the browser cache, no matter the login method</param>
+        public void Logout(bool forceClearCache = false)

```

Comment:
@jerriep Yeah, you're right. Stupid me. My thinking was wrong. Best would be to just do this:
````
public void Logout(bool clearCache)
        {
            this.CurrentUser = null;
            if(clearCache)
                WebBrowserHelpers.ClearCache();
        }
````
And
````
 public void Logout()
        {
            Logout(this.clearCache);
        }
````
This way, the user can leave the clearing of the cache up to the library, or force whether to clear it or not, unregarded of the login mechanism chosen.